packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/stop-bot-modal.tsx

```diff
@@ -9,6 +9,7 @@ type TStopBotModal = {
     is_dialog_open: boolean;
     onOkButtonClick: () => void;
     toggleStopBotDialog: () => void;
+    loadDataStrategy: () => void;
```

Comment:
can we keep it consistent and stick with one approach only??
at some places we are having VoidFunction and some places () => void;

Let's make it consistent if possible @maryia-matskevich-deriv

Reply:
@sandeep-deriv done